doc/index.rst

```diff
@@ -0,0 +1,23 @@
+.. dbxincluder documentation master file, created by
+   sphinx-quickstart on Wed May  4 10:26:24 2016.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to dbxincluder's documentation!
+=======================================
+
+Contents:
+
+.. toctree::
+   :maxdepth: 2
+
+   overview
+   usage
+   apidoc
```

Comment:
I think we can improve the structure, but for the time being, it's ok. For example, I'm missing the manpage here.

Let's do that in the next iteration, I'll open a new issue for that.